dbt_project.yml

```diff
@@ -1,13 +1,14 @@
 name: 'shopify_source'
-version: '0.11.0'
+version: '0.12.0'
 config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 models:
   shopify_source:
     +schema: stg_shopify
-    +materialized: table
-    tmp:
-      +materialized: view
+    +materialized: view
+    stg_shopify__metafield:
+      +materialized: table
```

Comment:
Can you explain why this model in particular is materialized as a table?

Is this because it doesn't have any downstream references?

Reply:
If this was intentional then we should call it out in the CHANGELOG

Reply:
Good point. I did this because it is used in all of the metafield models in the transform. But this makes me think, and since these are turned off by default, it doesn't make sense to always make this a table. I have adjusted to only make as a table if the enabled variable is set to true.

Reply:
Summarizing our chat from standup, this was for the benefit for redshift, but the logic was more complex than downstream fixes, so table was the most straightforward solution.